.github/no-response.yml

```diff
@@ -0,0 +1,17 @@
+# Configuration for no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
+# Label requiring a response
+# TODO: also close `needs-reproduction` issues (blocked by https://github.com/probot/no-response/issues/11)
+responseRequiredLabel: more-information-needed
+# Comment to post when closing an issue due to lack of response. 
+closeComment: >
+  Thank you for your issue!
+  
+  We haven’t gotten a response to our questions above.
+  With only the information that is currently in the issue,
+  we don’t have enough information to take action.
+  I’m going to close this but don’t hesitate to reach out
```

Comment:
Let's switch the "I'm" for a "We're" here, so it's phrased more like the team taking action and less a robot taking action 🙅‍♂️ 🤖

Reply:
✅ Changed